pkg/entities/record.go

```diff
@@ -104,9 +104,10 @@ func (b *baseRecord) GetOrderedElementList() []InfoElementWithValue {
 }
 
 func (b *baseRecord) GetInfoElementWithValue(name string) (*InfoElementWithValue, int, bool) {
-	for i, element := range b.orderedElementList {
+	for i := range b.orderedElementList {
+		element := &b.orderedElementList[i]
```

Comment:
this makes sense. 👍 
Do we have other places affecting every record that we can fix?

Reply:
I did not see it in other critical places where we iterate over the ordered element list. 
There are a couple of other non-critical places.